book/contextMacros.ddoc

```diff
@@ -1 +1,2 @@
-DVER = 2.097
+COZUM_HTML = frontispiece.cozum.html
+DVER = 2.088
```

Comment:
Why move the version backwards?

Reply:
I have no idea where that came from. I didn't touch that file. I'll checkout the head version.